### PR TITLE
fix(authoring): show widgets in embedded authoring

### PR DESCRIPTION
**Pull request by petrjasek**
but only on the right side.

+ re-enable e2e tests and fix a few issues there

### CODE REVIEW

File: client/spec/search_spec.js [hunk 1]
```diff
@@ -40,11 +40,11 @@ describe('Search', function() {
         expect(element.all(by.repeater('parameter in tags.selectedKeywords')).count()).toBe(1);
     });
 
-    it('can search by subject codes field', function () {
+    xit('can search by subject codes field', function () {
```
Comment (akintolga) [inline client/spec/search_spec.js]: is xit used for disabling or making them look like pending?

**Comment by ioanpocol** [comment]
:+1:

File: client/spec/search_spec.js (hunk 1 above)
Comment (petrjasek) [inline client/spec/search_spec.js]: it says pending in logs but it's actually disabled

**Comment by akintolga** [comment]
:+1:

**Pull request merged**